shardingsphere-jdbc/shardingsphere-jdbc-spring/shardingsphere-jdbc-spring-infra/shardingsphere-jdbc-spring-boot-starter-infra/src/main/java/org/apache/shardingsphere/spring/boot/schema/SchemaNameSetter.java

```diff
@@ -40,6 +42,10 @@
      */
     public static String getSchemaName(final Environment environment) {
         StandardEnvironment standardEnv = (StandardEnvironment) environment;
-        return standardEnv.getProperty(PREFIX + SCHEMA_NAME);
+        String schemaName = standardEnv.getProperty(PREFIX + SCHEMA_NAME);
+        if (StringUtils.isEmpty(schemaName)) {
```

Comment:
Please use `Strings`, just keep consist with other usage

Reply:
Is it passable to use ternary operator?

Reply:
> Is it passable to use ternary operator?

yes